Sources/SnapshotPreviewsCore/ConformanceLookup.swift

```diff
@@ -8,107 +8,109 @@
 import Foundation
 import MachO
 
-private func getTypeName(descriptor: UnsafePointer<TargetModuleContextDescriptor>) -> String? {
```

Comment:
Rearranged in order of public -> private and wrapped the `getPreviewTypes()` function in a `ConformanceLookup` class.